lib/components/flare_component.dart

```diff
@@ -58,4 +58,11 @@ class FlareComponent extends PositionComponent {
       _flareAnimation.height = height;
     }
   }
+
+  @override
```

Comment:
here you should do someting similar to what width and height are done, and set the value to the super.angle as well.

And use that value to set the angle on the animation when it loads here: https://github.com/flame-engine/flame/blob/f5538cc906e041e444d97246ceb84a7ff8aafb29/lib/components/flare_component.dart#L19